internal/datastore/postgres/common/pgx.go

```diff
@@ -88,6 +89,13 @@ func ConfigurePGXLogger(connConfig *pgx.ConnConfig) {
 			if level == pgx.LogLevelInfo {
 				level = pgx.LogLevelDebug
 			}
+			// do not log cancelled queries as errors
+			if errArg, ok := data["err"]; ok {
+				err, ok := errArg.(error)
+				if ok && (errors.Is(err, context.Canceled)) {
```

Comment:
are the parens necessary arounbd the `errors.Is`?

Reply:
they are not